src/components/accounts/SetRecoveryInfo.js

```diff
@@ -1,17 +1,22 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { isValidPhoneNumber } from 'react-phone-number-input'
-
+import { validateEmail } from '../../utils/account';
 import AccountFormSection from './AccountFormSection'
 import AccountFormContainer from './AccountFormContainer'
-import SetRecoveryInfoForm from './SetRecoveryInfoForm'
+import SetRecoveryInfoFormPhone from './SetRecoveryInfoFormPhone'
+import SetRecoveryInfoFormEmail from './SetRecoveryInfoFormEmail'
 import { requestCode, setupAccountRecovery, redirectToApp, clear, clearCode } from '../../actions/account';
 
 class SetRecoveryInfo extends Component {
    state = {
       loader: false,
       phoneNumber: '',
+      email: '',
+      validEmail: false,
+      sentEmail: false,
       isLegit: false,
+      BackupWithEmail: true
```

Comment:
why is this capitalized differently than everything else?

Reply:
Mistake from my side - will fix